index.js

```diff
@@ -176,6 +191,7 @@ Please generate tags using 'ember generate tag your-tag-name'`);
         'image',
         'imageMeta',
       ],
+      references: [{ name: 'posts', type: 'contents' }],
```

Comment:
Would it be valuable to include non custom-typed refs to demonstrate that string refs still work in combination?

Reply:
in terms of it being valuable, we should for sure include examples in the docs for broccoli-static-site-json but this is the type of relationship that we need for this type in empress-blog ;) there are plenty of other examples in this file of non custom-typed refs if we are pointing people to empress-blog to see working examples 👍